transaction-api/src/main/java/com/example/auction/transaction/api/TransactionEvent.java

```diff
@@ -5,7 +5,7 @@
 public abstract class TransactionEvent {
 
     private TransactionEvent() {}
-
+    
```

Comment:
It looks like some extraneous whitespace changes snuck in.

Here are some instructions for configuring Eclipse to strip trailing whitespace on save: http://stackoverflow.com/a/14178860